From 02361470f034c1457dddfe2cebe9bbd545a5bf40 Mon Sep 17 00:00:00 2001
From: Takashi Sakamoto <o-takashi@sakamocchi.jp>
Date: Sun, 17 Sep 2023 11:52:28 +0900
Subject: [PATCH] hwdb: ieee1394-unit-function: add Weiss Engieering MAN301

Michele Perrone contributes configuration ROM image for Weiss MAN301, which
includes two units for 1394TA AV/C devices 1.0 and TCAT general protocol.

* https://lore.kernel.org/alsa-devel/20230822135807.GA470519@workstation.local/

This commit adds hwdb entry for the model.

(cherry picked from commit 56f62f6f5a4dde21f09bf27003cf3522de4d0a4e)

Related: RHEL-5950
---
 hwdb.d/80-ieee1394-unit-function.hwdb | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/hwdb.d/80-ieee1394-unit-function.hwdb b/hwdb.d/80-ieee1394-unit-function.hwdb
index bca410495a..d62a2dec4b 100644
--- a/hwdb.d/80-ieee1394-unit-function.hwdb
+++ b/hwdb.d/80-ieee1394-unit-function.hwdb
@@ -1399,3 +1399,19 @@ ieee1394:ven000001F2mo*sp000001F2ver00000024
  ID_VENDOR_FROM_DATABASE=MOTU
  ID_MODEL_FROM_DATABASE=MOTU V4HD
  IEEE1394_UNIT_FUNCTION_VIDEO=1
+
+# Weiss Engineering MAN301 (DICE application): unit 0: AV/C devices 1.0.
+ieee1394:node:ven0x001c6amo0x00000bunits*0x00a02d:0x010001*
+ieee1394:ven00001C6Amo0000000Bsp0000A02Dver00010001
+ ID_VENDOR_FROM_DATABASE=Weiss Engineering Ltd.
+ ID_MODEL_FROM_DATABASE=MAN301
+ IEEE1394_UNIT_FUNCTION_AUDIO=1
+ IEEE1394_UNIT_FUNCTION_VIDEO=0
+
+# Weiss Engineering MAN301 (DICE application): unit 1: TCAT general protocol.
+ieee1394:node:ven0x001c6amo0x00000bunits*0x001c6a:0x000001*
+ieee1394:ven00001C6Amo0000000Bsp00001C6Aver00000001
+ ID_VENDOR_FROM_DATABASE=Weiss Engineering Ltd.
+ ID_MODEL_FROM_DATABASE=MAN301
+ IEEE1394_UNIT_FUNCTION_AUDIO=1
+ IEEE1394_UNIT_FUNCTION_VIDEO=0
